fw-child/resources/scss/_template.scss

```
@@ -3,7 +3,43 @@
 //
 
 #floating-header {
-  background-color: transparentize($dark, 0.1);
+  transition: transform 300ms linear, background-color 300ms linear;
+  transform-origin: top left;
+
+  @include media-breakpoint-down(md) {
+    transform: scale(1.5);
+  }
+
+  @include media-breakpoint-down(sm) {
+    transform: scale(1.33333);
+  }
+
+  @include media-breakpoint-up(md) {
+    background-color: transparentize($dark, 0.1);
+  }
+
+  .scrolled & {
+    transform: scale(1);
+    background-color: transparentize($dark, 0.1);
+  }
+
+}
+
+#menu-trigger {
+  transition: transform 300ms linear;
+  transform-origin: center;
+
+  @include media-breakpoint-down(md) {
+    transform: scale(.66667);
+  }
+
+  @include media-breakpoint-down(sm) {
+    transform: scale(.75);
+  }
+
+  .scrolled & {
```

Same comment as previously:
```suggestion
  body.scrolled & {
```